io.github.mpc_qt.Mpc-Qt.appdata.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I have no use for it upstream. I was going to leave it in the repo here and update it every time I make a release. Unless there's something I'm missing?

Reply:
it a standard for desktop app, it's not specific to flathub. So it should be managed upstream.

Reply:
Done, but I'll have to make a new release. For now I think I'll close this PR and we wait a bit while more work worth of a release gets done.

Reply:
it would be fine for this release. The idea as the in the future this repo only contains the manifest. Sometime it's not possible, but when it is, it is good.

Reply:
Thanks for the offer, but I'm one of those annoying people who likes to get it done right the first time. I think we'll be seeing you soon though.